src/components/LoaderButton/index.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import LoaderButton from './index';
+
+test('renders without loading', () => {
+  const wrapper = shallow(<LoaderButton />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+test('renders with loading', () => {
+  const wrapper = shallow(<LoaderButton isLoading={true} />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+test('renders without disabled', () => {
+  const wrapper = shallow(<LoaderButton disabled={false} />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+test('renders with disabled', () => {
+  const wrapper = shallow(<LoaderButton disabled={true} />);
+  expect(wrapper).toMatchSnapshot();
+});
```

Comment:
Don't depend on `toMatchSnapshot` heavily. All of these tests seem similar with little useful information.
Maybe you should check more about state of component. Or what's behavior of component could be different in case you pass different prop value, etc.